### ISSUE TITLE
`medaka train` seems to require a model

### ISSUE THREAD

**Issue by bjreisman** [nanoporetech/medaka#162]
**Bug Description**
I'm trying to use `medaka train` to train my own medaka model for our organisms, but keep running into the same error, even when using the walkthrough dataset. All the other steps until this point (alignment, features creation), and the medaka consensus module works fine as well. 

**Logging**
```
(medaka) root@db4156853e4a:/media/medaka_walkthrough# medaka train ${TRAINFEATURES} --train_name ${TRAINNAME} --epochs 10           
Traceback (most recent call last):
  File "/medaka/venv/bin/medaka", line 11, in <module>
    load_entry_point('medaka==1.0.1', 'console_scripts', 'medaka')()
  File "/medaka/venv/lib/python3.5/site-packages/medaka-1.0.1-py3.5-linux-x86_64.egg/medaka/medaka.py", line 642, in main
    args.model = medaka.models.resolve_model(args.model)
  File "/medaka/venv/lib/python3.5/site-packages/medaka-1.0.1-py3.5-linux-x86_64.egg/medaka/models.py", line 27, in resolve_model
    if os.path.exists(model):  # model is path to model file
  File "/medaka/venv/lib/python3.5/genericpath.py", line 19, in exists
    os.stat(path)
TypeError: stat: can't specify None for path argument
(medaka) root@db4156853e4a:/media/medaka_walkthrough# 
```
**Environment (if you do not have a GPU, write No GPU):**
 - Installation method: from github source
 - OS: Ubuntu 16.04 docker container (also have tried running it without docker on Ubuntu 20.04 with the same issue)
 - GPU model: Quadro P2200
 - Nvidia driver version: 425.21
 - CUDA version 10.1
 - cuDNN version 7.6.5

**Additional context**
If I run it with a model, it does seem to start: (showing first and last few lines)
```
(medaka) root@db4156853e4a:/media/medaka_walkthrough# medaka train ${TRAINFEATURES} --train_name ${TRAINNAME} --epochs 10 --model r941_min_high_g351
[21:15:09 - root] The path training exists. Results will be overwritten.
[21:15:09 - DataIndex] Loaded 1/1 (100.00%) sample files.
[21:15:09 - TrainBatcher] Sample features have shape (1000, 10)
[21:15:09 - TrainBatcher] Randomly selected 1625 (20.00%) of features for validation (seed None)
[21:15:09 - TrainBatcher] Got 6497 samples in 64 batches (6497000 labels) for training
[21:15:09 - TrainBatcher] Got 1625 samples in 16 batches (1625000 labels) for validation
...
Root Member(assigned_device_name_index_=-1 requested_device_name_='/device:GPU:0' assigned_device_name_='' resource_device_name_='/device:GPU:0' supported_device_types_=[CPU, XLA_CPU, XLA_GPU] possible_devices_=[]
AssignAddVariableOp: CPU XLA_CPU XLA_GPU 
ReadVariableOp: CPU XLA_CPU XLA_GPU 
AssignVariableOp: CPU XLA_CPU XLA_GPU 
VarIsInitializedOp: CPU XLA_CPU XLA_GPU 
VarHandleOp: CPU XLA_CPU XLA_GPU 
Const: CPU XLA_CPU XLA_GPU 

Colocation members, user-requested devices, and framework assigned devices, if any:
  count_1/Initializer/zeros (Const) 
  count_1 (VarHandleOp) /device:GPU:0
  count_1/IsInitialized/VarIsInitializedOp (VarIsInitializedOp) /device:GPU:0
  count_1/Assign (AssignVariableOp) /device:GPU:0
  count_1/Read/ReadVariableOp (ReadVariableOp) /device:GPU:0
  metrics/qscore/AssignAddVariableOp_1 (AssignAddVariableOp) /device:GPU:0
  metrics/qscore/ReadVariableOp_1 (ReadVariableOp) /device:GPU:0
  metrics/qscore/div_no_nan/ReadVariableOp_1 (ReadVariableOp) /device:GPU:0
  VarIsInitializedOp_14 (VarIsInitializedOp) /device:GPU:0

Epoch 1/10
```
but judging by the CPU/GPU utilization it doesn't seem to actually be doing anything (probably related to this bug: https://github.com/nanoporetech/medaka/issues/86)

**Comment by cjw85** [comment]
Hi @bjreisman,

This is a newly introduced bug and not related to #86. We will fix it forthwith!

**Comment by cjw85** [comment]
This should be fixed in v1.0.2 available now.

**Comment by bjreisman** [comment]
Indeed it is! Thanks for the fix.
